### ISSUE TITLE
EAC stuck on "Initialize" - Working well on the deb version

### ISSUE THREAD

**Issue by NoksC** [canonical/steam-snap#198]
**Describe the bug**

When launching a game that require EAC get stuck on EAC splash screen "initialize".

**To Reproduce**

Install and try to launch "Insurgency Sandstorm" for exemple.

**Expected behavior**

EAC load fine, the game launch fine, and the multiplayer is playable.

**Environment**

Ubuntu 23.04 Under wayland with AMD GPU. Can't see nothing in terminal explain that.

**Additional Context**

This same game with EAC work well on the regular deb version.

**Comment by HandsomeSimon** [comment]
I've also seen this issue with BattleBit Remastered. 
Are these any logs that we can collect for this? Running Steam from the Terminal doesn't seem to display anything useful

**Comment by Giako** [comment]
Same happens for Brawlhalla, work-around is to start the game with the `-noeac` command line option.

**Comment by ErickLandowski** [comment]
Happens to me as well with Battlebit and Armored Core 6. Both games use EAC.

**Comment by AussieAnon** [comment]
Same issue here. Problems with all my EAC games like Battlebit, 7 days to die etc etc. Hope this gets fixed soon

**Comment by lexasok** [comment]
The same: Ubuntu, Hunt Showdown

**Comment by NoksC** [comment]
Anti-cheat specific but the same anti-cheat with the same game work well on deb and flatpak steam version.

What's the point to make a snap if isn't for, at least, have the same experience we have on the deb package ?. You deleted the "Working well on the deb version" of my title but its realy the case.

**Comment by ashuntu** [comment]
It's assumed that every issue in this repository affects the Snap only (or is something the Snap can provide/fix itself), I removed that part of the title to simplify it for readability’s sake. Sorry if doing that was misleading, concise titles and labels just make issues easier to find and track

**Comment by ashuntu** [comment]
If not already, this may work now with 2.64 snapd. You can try by refreshing snapd with `snap refresh snapd --edge` (and restarting). **Make sure to go back to stable after testing** with `snap refresh snapd --stable`.

**Comment by Giako** [comment]
Yes, confirmed. Brawlhalla with EAC enabled works with snapd refreshed at `edge` version 👍🏻
